package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Blockly Developer Tools",
+  "main": "index.html",
+  "window": {
+    "width": 1500,
+    "height": 1000
+  },
+  "dependencies": {
+    "jquery": "3.2.1",
+    "babel": "4.7.16"
```

Comment:
I don't see babel being used anywhere, nor do I think we'll need it in a node environment.